samples/secure-fraud-detection/README.md

```diff
@@ -0,0 +1,127 @@
+# Secure Fraud Detection Demo
+
+This demo showcases the implementation of a simple fraud detection system using LLMs (GPT-4 in this case) for users authenticated with Google.
```

Comment:
AFAIU it's configured to use `gpt-3.5-turbo` (the default, unless you override `quarkus.langchain4j.openai.chat-model.model-name`)

Reply:
Correct

Reply:
I haven't started working on README yet, thanks for pointing it out though, will keep it in mind